examples/README.md

```diff
@@ -13,20 +13,30 @@
 
 # Examples of Enceladus usage
 ## Custom Rules Implementations and usage
-### CustomRuleSample1
+### Requirements
+* Spark 2.4+ (might work with earlier, not tested)
```

Comment:
Definitely works with 2.2.1, which is the spark we are using in our environments. Also, I tested your code on my machine running 2.2.1.